utest/api/test_run_and_rebot.py

```diff
@@ -90,6 +90,14 @@ def test_multi_options_as_single_string(self):
         assert_equals(run_without_outputs(self.data, exclude='fail'), 0)
         self._assert_outputs([('FAIL', 0)])
 
+    def test_listeners_unregistration(self):
+        module_file = join(ROOT, 'utest', 'resources', 'Listener.py')
+        assert_equals(run(self.data, outputdir=TEMP, report='none', listener=[module_file+":1"]), 1)
```

Comment:
Is `run` needed here instead of `run_without_outputs` that's used by above tests?